.github/workflows/deploy.backend-api.yml

```diff
@@ -0,0 +1,116 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+
+name: Build and Deploy Backend API
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  AZURE_WEBAPP_PACKAGE_PATH: 'app/Monitor'
+  DOTNET_VERSION: '8.x'
+  DATA_PROJECT: ./app/Data/Data.csproj
+  WEB_APP_PROJECT: ./app/Monitor/Monitor.csproj
+  DOTNET_BUILD_CONFIG: release
+
+jobs:
+  # Publish Db Migrations 
+  publish_DbMigrations:
+    name: Publish Database Migrations Bundle
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: ${{ env.DOTNET_VERSION }}
+
+      - name: Restore repo dotnet tools
+        run: dotnet tool restore
+        shell: bash
+
+      - name: Build bundle
+        run: dotnet ef migrations bundle -p ${{ env.WEB_APP_PROJECT }} -o ${{ github.workspace }}/efbundle -r linux-x64 --self-contained --configuration ${{ env.DOTNET_BUILD_CONFIG }}
+        shell: bash
+
+      - name: Publish migrations bundle artifact
+        uses: actions/upload-artifact@v4
+        with:
+          path: ${{ github.workspace }}
+          name: migrations
+
+  # Publish Web App
+  publish_webapp:
+    name: Publish Web App
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Get Git Commit Hash
+        id: vars
+        run: |
+          calculatedSha=$(git rev-parse --short ${{ github.sha }})
+          echo "GitHash=$calculatedSha" >> $GITHUB_ENV
+
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: ${{ env.DOTNET_VERSION }}
+      
+      - name: dotnet publish
+        run: dotnet publish ${{ env.WEB_APP_PROJECT }} -c ${{ env.DOTNET_BUILD_CONFIG }} -o ${{ github.workspace }} -p:GitHash=${{ env.GitHash }}
+        shell: bash
+
+      - name: Publish webapp artifact
+        uses: actions/upload-artifact@v4
+        with:
+          path: ${{ github.workspace }}
+          name: webapp
+
+  # Deploy UAT
```

Comment:
Once UAT is completely working - you'll need to add the production environment also.